src/util/zeit-agent.js

```diff
@@ -0,0 +1,53 @@
+import {parse as parseUrl} from 'url';
+import qs from 'querystring';
+import nodeFetch from 'node-fetch';
+import Agent from './agent';
+
+export default class ZeitAgent extends Agent {
```

Comment:
This is simple utility where we can talk to our APIs easily.

Reply:
Why do we need this We already have an Agent that works just fine? What's the difference?

I think we shouldn't add a new one because the old one is also a ZEIT Agent – we don't talk to other services than ZEIT services.

Reply:
The current Agent is a simple HTTP2 agent. It doesn't handle take care of setting the `teamId` param and pretty simple things.

So, this is a subclass which understand our API and handle it. I don't mind moving these methods to the main class, but I think this approach is better.

Reply:
@leo yeah I saw that.
I don't like repeating when we have a clear path to move the common logic into class, utitlity function, etc.
That's less error prone and easy to write code.